templates/python-package.yml

```diff
@@ -33,16 +33,10 @@ jobs:
     displayName: 'Install dependencies'
 
   - script: |
-      pip install pytest
-      pytest tests --doctest-modules --junitxml=junit/test-results.xml
+      pip install pytestpytest-azurepipelines
```

Comment:
`pytest-azurepipelines` ?

Reply:
Python is truly a whitespace-significant language ;)